lib/iex/test/test_helper.exs

```diff
@@ -1,6 +1,10 @@
+assert_timeout = String.to_integer(
+  System.get_env("ELIXIR_ASSERT_TIMEOUT") || "500"
```

Comment:
Nitpick, is everything on one line too long here? It doesn't look too long, the `ExUnit.start/1` call below should be roughly the same?